QUEST_LV_0200.tsv

```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
+QUEST_LV_0200_20150323_005240	$Ask for help from the Pyromancer Master at Klaipeda
+QUEST_LV_0200_20150323_005241	$Gitis told you that the workers at the settlement land were attacked by the monsters and they are sick. Talk to the Pyromancer Master to get her help.
+QUEST_LV_0200_20150323_005242	$The Pyromancer Master told you that he will definitely help you. Talk to the Pyromancer Master again.
 QUEST_LV_0200_20150323_005243	Collect the samples of pink Chuparukas at the settlement land
 QUEST_LV_0200_20150323_005244	{memo X}Pyromancer Master told you that he will definitely help you. First get some samples of pink Chuparukas at the settlement land.
-QUEST_LV_0200_20150323_005245	Hand it over to Pyromancer Master
-QUEST_LV_0200_20150323_005246	You've obtained the samples of pink Chuparukas. Hand them over to Pyromancer Master.
+QUEST_LV_0200_20150323_005245	$Hand it over to the Pyromancer Master
+QUEST_LV_0200_20150323_005246	$You've obtained the samples of pink Chuparukas. Hand them over to the Pyromancer Master.
 QUEST_LV_0200_20150323_005247	Obtain %s from the settlement land
-QUEST_LV_0200_20150323_005248	Pyromancer Master who was investigating the samples of the monsters doesn't feel good about something. Talk to Pyromancer Master.
+QUEST_LV_0200_20150323_005248	$The Pyromancer Master who was investigating the samples of the monsters doesn't feel good about something. Talk to the Pyromancer Master.
 QUEST_LV_0200_20150323_005249	Search for the clues of the massive chaos at the settlement land
-QUEST_LV_0200_20150323_005250	Pyromancer Master can feel some unknown poison and the spell. Go back to the settlement land and find out the true nature of the poison and the spell.
-QUEST_LV_0200_20150323_005251	You've found a leather and a dagger with the magic field drawn in them at Nuodo Bank. Take them to Pyromancer Master.
+QUEST_LV_0200_20150323_005250	$The Pyromancer Master can feel some unknown poison and the spell. Go back to the settlement land and find out the true nature of the poison and the spell.
+QUEST_LV_0200_20150323_005251	$You've found a leather and a dagger with the magic field drawn in them at Nuodo Bank. Take them to the Pyromancer Master.
 QUEST_LV_0200_20150323_005252	Destroy the unknown altar
-QUEST_LV_0200_20150323_005253	Unfortunately, Pyromancer Master doesn't know well about the spell. Talk to Pyromancer Master again.
-QUEST_LV_0200_20150323_005254	Talk to Bokor Master about the true nature of the spell.
-QUEST_LV_0200_20150323_005255	Bokor Master knows well about the spells. Ask Bokor Master about the true nature of the spells.
-QUEST_LV_0200_20150323_005256	Bokor Master told you that the spell spreads the poison that is on the the dagger. But, he told you that he doesn't know well about the dagger so told you to meet Ranger Master.
-QUEST_LV_0200_20150323_005257	Talk to Gitise
+QUEST_LV_0200_20150323_005253	$Unfortunately, the Pyromancer Master doesn't know well about the spell. Talk to the Pyromancer Master again.
```

Comment:
it's a dagger with a spell cast on it. for consistency's sake 'spell' is probably better.